addons/docs/docs/docspage.md

```diff
@@ -0,0 +1,198 @@
+<center>
+  <img src="./media/docspage-hero.png" width="100%" />
+</center>
+
+# Storybook DocsPage
+
+When you install [Storybook Docs](../README.md), `DocsPage` is the zero-config default documentation that all stories get out of the box. It aggregates your stories, text descriptions, docgen comments, props tables, and code examples into a single page for each component.
+
+- [Motivation](#motivation)
+- [Component parameter](#component-parameter)
+- [DocsPage slots](#docspage-slots)
+- [More resources](#more-resources)
+
+## Motivation
+
+`DocsPage` is the successor to [`addon-info`](https://github.com/storybookjs/storybook/tree/next/addons/info), which was one of the most popular Storybook addons despite many limitations.
+
+Like `addon-info`, `DocsPage` provides sensible defaults, meaning it adds documentation to your existing Storybook without requiring any additional work on your part.
+
+However, `DocsPage` brings the following improvements:
+
+- It supports all frameworks that Storybook supports, including React, Vue, Angular and [many others](../README.md#framework-support).
+- It generates better documentation that can be used as a standalone docs site, independently of Storybook.
+- It supports better configuration, so you can capture project specific information with ease.
+- It's built to work with [`MDX`](./mdx.md`) when you need more control of your documentation.
+
+## Component parameter
+
+`DocsPage` pulls info from many sources, but one of the main ones is the `component` parameter, which is a new addition to Storybook in 5.2. It's based on the idea that each component should have an associated set of documentation and stories (versus organizing it in some other way).
```

Comment:
"It's based on the idea" --> "It's based on best practices from the Storybook community-at-large"